conda/post-link.sh

```diff
@@ -0,0 +1,33 @@
+
+echo "post-link.sh, PREFIX: $PREFIX"
+
+mv $PREFIX/bin/cyclus $PREFIX/bin/cyclus_base
+echo "
+#!/bin/bash
+export LD_LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib/cyclus
+export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib/cyclus
+export DYLD_LIBRARY_PATH=
+export CYCLUS_NUC_DATA=$PREFIX/share/cyclus/cyclus_nuc_data.h5
+export CYCLUS_PATH=$PREFIX/lib/cyclus
+export CYCLUS_RNG_SCHEMA=$PREFIX/share/cyclus/cyclus.rng.in
+
+$PREFIX/bin/cyclus_base \$*
+
+" > $PREFIX/bin/cyclus
+chmod 755 $PREFIX/bin/cyclus
+
+mv $PREFIX/bin/cyclus_unit_tests $PREFIX/bin/cyclus_unit_tests_base
+echo "
+#!/bin/bash
+export LD_LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib/cyclus
+ export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib/cyclus
```

Comment:
not just "recipe" - that could be confused with our other fuel cycle domain
use of the word.

On Tue, Apr 28, 2015 at 3:18 PM, Anthony Scopatz <notifications@github.com>
wrote:

> In conda/post-link.sh
> <https://github.com/cyclus/cyclus/pull/1136#discussion_r29289754>:
>
> > +export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib/cyclus
> > +export DYLD_LIBRARY_PATH=
> > +export CYCLUS_NUC_DATA=$PREFIX/share/cyclus/cyclus_nuc_data.h5
> > +export CYCLUS_PATH=$PREFIX/lib/cyclus
> > +export CYCLUS_RNG_SCHEMA=$PREFIX/share/cyclus/cyclus.rng.in
> > +
> > +$PREFIX/bin/cyclus_base \$*
> > +
> > +" > $PREFIX/bin/cyclus
> > +chmod 755 $PREFIX/bin/cyclus
> > +
> > +mv $PREFIX/bin/cyclus_unit_tests $PREFIX/bin/cyclus_unit_tests_base
> > +echo "
> > +#!/bin/bash
> > +export LD_LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib/cyclus
> > + export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib/cyclus
>
> weird, leading space.... never noticed that before.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus/pull/1136/files#r29289754>.
>